### ISSUE TITLE
Proposed template for onnx models readme

### ISSUE THREAD

**Issue by mx-iao** [onnx/models#27]
Hi ONNX team, I'm working with @prasanthpul to standardize on a README template for the models in the zoo. Please let me know your thoughts. See [here](https://github.com/onnx/models/tree/master/mnist) for the MNIST model for reference.

# Model name
Download: link to model download
Model size: x MB
## Description
Description of what the model does
### Paper
Name and link to the paper that the model implements, if applicable.
### Dataset
Dataset that was used to train the model
## Source
Reference to the tutorial used for training and/or the source/Github repo of the code that the ONNX model was generated or converted from (e.g. TinyYOLO CoreML).  
If this is different than the original model code/source (e.g. TinyYOLO DarkNet), provide a link to that, too.
### Input 
Expected model input format - type and shape
### Output
Expected model output format - type and shape
### Pre-processing steps
Description of what pre-processing steps need to be done on the data before feeding them to the model
### Post-processing steps
If applicable, description or link to reference of any post-processing steps to perform on model output
### Sample test data
Short description on the included sample test data (.pb or .npz).
## Results/accuracy on test set
If possible, to motivate model reproducibility and integrity.
## License

**Comment by michaelschwier** [comment]
This would be massively helpful. Especially a precise specification of the input (incl. preprocessing) and output.

**Comment by michaelschwier** [comment]
Hi @mx-iao,
Yes, that is a good example. I would say it is really important to have a precise definitions of input (incl. preprocessing) and output format for all models in the zoo, to make them easily reusable.

My current issue - and why I would be happy to have such a definition for all models: I am working with the squeezenet from this repo. I got an older version of it running and now updated to the newer version which seems to use a different normalization/range for the input as well as for the output. I kinda figured out smth that works (image - 127.5 for input, output is already softmaxed), but I cannot be sure if this is correct or just smth close enough to the expected preprocessing/ranges to give reasonable results.

**Comment by mx-iao** [comment]
Thanks for your feedback @michaelschwier. Once we standardize on a README template we will go back and update all the models that are missing this information.

**Comment by michaelschwier** [comment]
Sounds great thx. 
Is there anyone who can give me details about the squeezenet in the meantime?

**Comment by houseroad** [comment]
This is great!

Here are some additional field we should provide:
1) unique name of the model
2) model source (where the model is converted/exported, maybe provide the original model link?)
3) input/output shape/type information

**Comment by houseroad** [comment]
For the model itself, we may need to provide multiple versions.

**Comment by postrational** [comment]
You should also provide a md5 checksum or some other way of telling if the model I have on my disk is current. This is especially true of models which only exist as links to S3 or another location which is not under version control.

**Comment by NiklasGustafsson** [comment]
I suggest that we include a model version in the repo's path to the model, and that the version number should be described in the README.

**Comment by prasanthpul** [comment]
template has been in use for some time
